modules/custom/activity_logger/activity_logger.tokens.inc

```diff
@@ -28,6 +28,10 @@ function activity_logger_token_info() {
     'description' => t("The related node title."),
   ];
 
+  $message['gtrimmed-title'] = [
```

Comment:
@rollki I would recommend not introduce a new token for group titles.
Maybe you can check and use `hook_tokens_alter()` in the dashboard module and trim the existing `gtitle` when the Dashboard module is enabled?

Reply:
As we discussed we don't want it yet, so I have removed changes related to the trimmed group title, in the future when it will be necessary we will be able to add this feature.